openassessment/xblock/test/data/update_xblock.json

```diff
@@ -56,59 +56,6 @@
         "editor_assessments_order": ["student-training", "peer-assessment", "self-assessment", "staff-assessment"]
     },
 
-    "no_staff_backwards_compatible": {
-        "criteria": [
-            {
-                "order_num": 0,
-                "name": "0",
-                "label": "Test criterion",
-                "prompt": "Test criterion prompt",
-                "options": [
-                    {
-                        "order_num": 0,
-                        "points": 0,
-                        "name": "0",
-                        "label": "No",
-                        "explanation": "No explanation"
-                    },
-                    {
-                        "order_num": 1,
-                        "points": 2,
-                        "name": "1",
-                        "label": "Yes",
-                        "explanation": "Yes explanation"
-                    }
-                ],
-                "feedback": "required"
-            }
-        ],
-        "prompts": [{"description": "My new prompt 1."}, {"description": "My new prompt 2."}],
-        "feedback_prompt": "Feedback prompt",
-        "feedback_default_text": "Feedback default text",
-        "submission_due": "4014-02-27T09:46",
-        "submission_start": "4014-02-10T09:46",
-        "title": "My new title.",
-        "file_upload_type": null,
-        "white_listed_file_types": null,
-        "allow_latex": false,
-        "leaderboard_show": 0,
-        "assessments": [
-            {
-                "name": "peer-assessment",
-                "must_grade": 5,
-                "must_be_graded_by": 3,
-                "start": null,
-                "due": "4014-03-10T00:00"
-            },
-            {
-                "name": "self-assessment",
-                "start": null,
-                "due": null
-            }
-        ],
-        "editor_assessments_order": ["student-training", "peer-assessment", "self-assessment"]
-    },
```

Comment:
See note above. No longer is it possible to update the xblock to an order that does not include `staff`. My reasoning is that staff will always be present, so let's force the user to acknowledge that they know that and have included it in their order.